src/util/namespace.h

```diff
@@ -9,12 +9,6 @@ class namespacet
 {
 public:
   virtual const symbolt *lookup(const irep_idt &name) const;
-  const symbolt *lookup(const irept &irep) const
-  {
-    return lookup(irep.identifier());
-  }
-
-  void follow_symbol(irept &irep) const;
```

Comment:
I don't understand the purpose of removing these 2 API functions here. The `lookup(irept)` one is definitely useful judging by the amount of `.identifier()` suffixes added throughout this PR. And could you explain why you're moving `follow_symbol()` out of namespacet?

Reply:
In summary, I don't want people to use it.

- follow symbol is only used by the old frontend
- by removing the lookup for irept it will be easier to my next PR which is replacing the symbol with symbolt2t in namespacet.